views/administration/forms/form-classes-manage.hbs

```diff
@@ -68,11 +68,11 @@
         {{#if schoolUsesLdap}}
             <strong>Schüler loggen sich in Anwesenheit eines Erziehungberechtigten ein</strong>
 
-            <p>Durch Anbindung an ein Landes- oder Schulsystem können sich die Schüler bereits in der {{theme.short_title}} einloggen. Fordere deine Schüler auf, den ersten Login im Beisein eines Erziehungsberechtigten vorzunehmen, damit dieser sein Einverständnis erklären kann und die Registrierung abschließen kann.</p>
+            <p> {{$t "administration.longText.text.provideStudentsAndTheirParents" (dict "shortTitle" theme.short_title)}}</p>
         {{else}}
             <strong>Lade Eltern und Schüler per Link ein, die Einverständniserklärung abzugeben</strong>
```

Comment:
missing interpolation

Reply:
setting the variable was wrong actually. It's solved. Thanks!